examples/rfm7xAndBk242xCompatiblity/rfm7x_hardware.cpp

```diff
@@ -0,0 +1,180 @@
+#if defined(ARDUINO)
```

Comment:
No license nor authorship. Also this file won't be available in the docs (we use Doxygen to generate the docs), this missing comment block also needs a `@file` command.

Much of the rest of this file is necessary if the libraries are combined properly.